### ISSUE TITLE
currentline is nil

### ISSUE THREAD

**Issue by glepnir** [notomo/vusted#8]
run `vusted tests` it works fine. all test case passed. run in github workflow I got this error

```
not ok 1 - 
...im/.luarocks/share/lua/5.1/busted/outputHandlers/TAP.lua:44: attempt to concatenate field 'currentline' (a nil value)
```

mine `ci.yml` is

```yml
  test:
    runs-on: ubuntu-latest
    name: test
    steps:
      - uses: rhysd/action-setup-vim@v1
        id: vim
        with:
          neovim: true
          version: nightly

      - name: luajit
        uses: leafo/gh-actions-lua@v8
        with:
          luaVersion: "luajit-2.1.0-beta3"

      - name: luarocks
        uses: leafo/gh-actions-luarocks@v4

      - name: run test
        shell: bash
        run: |
          luarocks install luacheck
          luarocks install vusted
          vusted tests
```

**Comment by notomo** [comment]
Thanks for reporting.

Currently, vusted seems to raise error when test cases are not found.

Would you confirm whether your actual ci.yml executes checkout action?

- example diff: https://github.com/notomo/vusted/commit/4742ba7
    - before: https://github.com/notomo/vusted/actions/runs/4877800085/jobs/8702813728#step:5:111
    - after: https://github.com/notomo/vusted/actions/runs/4877861466/jobs/8702942078#step:6:110

**Comment by notomo** [comment]
📝

> Currently, vusted seems to raise error when test cases are not found.

related?
https://github.com/lunarmodules/busted/issues/569

**Comment by glepnir** [comment]
right . missing checkout in this step. thx

**Comment by Julian** [comment]
Perhaps worth reopening this anyhow? It's at least a confusing error message which I just got it the first time trying out vusted (which I expected to have nothing happen essentially, as I haven't written any tests yet.)

**Comment by notomo** [comment]
📝created PR for busted: https://github.com/lunarmodules/busted/pull/731

**Comment by notomo** [comment]
fixed: notomo/vusted@c9e9048
